runtime/manifest.js

```diff
@@ -412,7 +412,7 @@ ${e.message}
         connection.tags = connectionItem.target ? connectionItem.target.tags : [];
         let direction = {'->': 'out', '<-': 'in', '=': 'inout'}[connectionItem.dir];
         if (connection.direction) {
-          if (connection.direction != direction && direction != 'inout') {
+          if (connection.direction != direction && direction != 'inout' && !(connection.direction == 'host' && direction == 'in')) {
```

Comment:
so do we enforce hosted particles to use <- arrow?
maybe we can have a reverse map to do the validation here? currently this if statement is a bit confusing.

Reply:
No, that was my mistake. We already allow connections to be more generic than the arguments that they bind to. So `=` becomes `<-`, but `->` is never valid here.

Reply:
I agree it's confusing, not sure about a map, it's not 1:1.